### ISSUE TITLE
Two open Windows/tabs in two different channels is not synchronous 

### ISSUE THREAD

**Issue by arnehorn** [be-gosu/issues#73]
When I have go.su open in two different tabs to have two Channels open, the channel list in terms of new messages is not updated correctly.

I have activity in #welcome doesn't trigger a 'no more new messages' on the other tab for the channel #welcome 

If you have any questions as me :)

**Comment by felixam** [comment]
Yes, there is no "read status" sync between user clients. Will probably solve that via "request to pull" websocket system message once we added that
